typescript/cli/src/commands/options.ts

```diff
@@ -25,13 +25,12 @@ export const outDirCommandOption: Options = {
 export const coreArtifactsOption: Options = {
   type: 'string',
   description: 'File path to core deployment output artifacts',
-  alias: 'ca',
+  alias: 'a',
```

Comment:
The single letter is better b.c. it prints nicer help text (and is a more common convention)